packages/frontend/component/src/components/auth-components/change-email-page.tsx

```diff
@@ -42,6 +44,7 @@ export const ChangeEmailPage = ({
     <AuthPageContainer
       title={t['com.affine.auth.change.email.page.title']()}
       subtitle={t['com.affine.auth.change.email.page.subtitle']()}
+      windowsAppControls={windowsAppControls}
```

Comment:
Will this affect the signup modal?

Reply:
I don't think it will affect other pages. Basically, places where the AffineOtherPageLayout component can be used cannot be directly jumped in the client. It seems that you have to open the browser.

Reply:
For example, this page
![CleanShot 2024-09-10 at 11 05 15@2x](https://github.com/user-attachments/assets/5afef3b4-b19b-4eaf-b7b9-9639c8e6f348)

Reply:
By the way, I adjusted the `404` page and `PageNotFound` components. I think it is more reasonable to call `AffineOtherPageLayout` only when jumping to the 404 page.